projects/packages/publicize/src/class-publicize-base.php

```diff
@@ -289,6 +289,22 @@ public function use_admin_ui_v1(): bool {
 			|| $this->has_connections_management_feature();
 	}
 
+	/**
+	 * Whether the user has the feature flag enabled.
+	 *
+	 * @param string $flag_name The feature flag to check. Will be prefixed with 'jetpack_social_use_' for the option.
+	 * @param string $plan_name The plan name to check for for the Current_Plan check, without the social- prefix.
+	 * @return bool
+	 */
+	public function use_feature_flag( $flag_name, $plan_name ): bool {
```

Comment:
Just a thought about naming...

Can we call this `has_feature` instead?

Also, the feature doesn't depend on the user as mentioned in the doc block. It rather depends on the site plan, doesn't it?

Reply:
If you decide to name it `has_feature`, we can then also update the option name to use `_has_` instead of `_use_`